tutorials/shading/visual_shader.rst

```diff
@@ -0,0 +1,101 @@
+.. _doc_visual_shader:
+
+Visual Shader
+=============
+
+Introduction
+------------
+
+VisualScript is an alternative for users that cannot code, Similarly, Visual Shader is the visual alternative for creating shaders.
```

Comment:
"Just as VisualScript is an alternative for users that cannot code, VisualShaders are the visual alternative for creating shaders."